<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(53,53,53,255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Book">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>200</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:Red;
font-size:15pt;
border-radius:7px
</string>
     </property>
     <property name="text">
      <string>Book</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ksdds">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;fgf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 7px;
font-size: 9pt;
color: black;
</string>
     </property>
     <item>
      <property name="text">
       <string>Pick-up Address</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icon/icons/pin_drop_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icon/icons/pin_drop_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Airport &quot;Manas&quot;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Office &quot;PrimeCar&quot;, Chokmorova 256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 7px;
font-size: 9pt;
color: black;
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <property name="minimumContentsLength">
      <number>6</number>
     </property>
     <item>
      <property name="text">
       <string>List of cars</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icon/icons/directions_car_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icon/icons/directions_car_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hyundai Solaris</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chevrolet Cobalt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Toyota Corolla</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hyundai Sonata</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Toyota Rav 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Subaru Forester</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KIA Seltos 2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lexus RX330</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Toyota 4Runner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Toyota Fortuner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Toyota Landcruiser 200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lexus LX570</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 7px;
font-size: 9pt;
color: black;
</string>
     </property>
     <item>
      <property name="text">
       <string>Return address</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icon/icons/pin_drop_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icon/icons/pin_drop_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Airport &quot;Manas&quot;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Office &quot;PrimeCar&quot;, Chokmorova 256</string>
      </property>
     </item>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 7px;
font-size: 9pt;
color: black;
</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 7px;
font-size: 9pt;
color: black;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>541</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(53,53,53,255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(44,55,68,255)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Снимок экрана 2025-04-10 102825.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/Screenshots/Снимок экрана 2025-04-10 103833.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Pictures/Screenshots/Снимок экрана 2025-04-10 104041.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:19pt;
font-weight:bold

</string>
      </property>
      <property name="text">
       <string>What class of car Do you need?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sedan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-radius: 7px;
font-size: 13pt;
color: black;
</string>
      </property>
      <property name="text">
       <string>Sedan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Crossover">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-radius: 7px;
font-size: 13pt;
color: black;
</string>
      </property>
      <property name="text">
       <string>Crossover</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SUV">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border-radius: 7px;
font-size: 13pt;
color: black;
</string>
      </property>
      <property name="text">
       <string>SUV</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:12pt;
color:white;
</string>
      </property>
      <property name="text">
       <string>For city</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:12pt;
color:white</string>
      </property>
      <property name="text">
       <string>For city/highway</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:12pt;
color:white</string>
      </property>
      <property name="text">
       <string>For travel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font-weight:bold;
font-size:23pt;

</string>
    </property>
    <property name="text">
     <string>CAR RENTAL IN BISHKEK</string>
    </property>
   </widget>
   <widget class="QFrame" name="welcomeLabel&#10;">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:DimGray</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:15pt;
border-radius:7px</string>
     </property>
     <property name="text">
      <string>+996773533969</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icon/icons/call_24dp_FFFFFF_FILL0_wght400_GRAD0_opsz24 (1).svg</normaloff>:/icon/icons/call_24dp_FFFFFF_FILL0_wght400_GRAD0_opsz24 (1).svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="Login">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 7px;
color: black;
</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Pictures/Screenshots/Снимок экрана 2025-04-10 121302.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="welcomeLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:10pt
</string>
     </property>
     <property name="text">
      <string>Hello, guest!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
